pyuploadcare/ucare_cli.py

```diff
@@ -145,6 +148,67 @@ def create_group(arg_namespace):
     pp.pprint(group)
 
 
+def sync_files(arg_namespace):
+    if not os.path.exists(arg_namespace.path):
+        os.makedirs(arg_namespace.path)
+
+    if arg_namespace.uuids:
+        files = (File(uuid) for uuid in arg_namespace.uuids)
+    else:
+        files = FileList()
+
+    for f in files:
+        url = f.info()['original_file_url']
+        local_filename = os.path.join(arg_namespace.path, f.filename())
```

Comment:
you could provide a pattern to build file path, like during remote copying :)

![selection_040](https://cloud.githubusercontent.com/assets/925309/10286236/7cff0b90-6b95-11e5-9ddb-581276a747f4.png)

if that's too much, the easiest would be just to have UUID as part of the path